tests/dummy_operator.py

```diff
@@ -18,6 +18,7 @@ class DummyOperator(TransportOperator):
 
     """
     def __init__(self):
+        self.prev_res = None
```

Comment:
Get rid of pass below this

Reply:
Was this pass just so that the method wouldn't be empty ?